examples/se/numberguess/src/main/java/org/jboss/weld/environment/se/example/numberguess/Main.java

```diff
@@ -0,0 +1,11 @@
+package org.jboss.weld.environment.se.example.numberguess;
+
+import org.jboss.weld.environment.se.Weld;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Weld weld = new Weld().beanClasses(Game.class, Generator.class, MessageGenerator.class, NumberGuessFrame.class).disableDiscovery();
```

Comment:
Why not `new Weld().beanClasses(Game.class, Generator.class, MessageGenerator.class, NumberGuessFrame.class).disableDiscovery().initialize();`?